service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandlerEmailTemplateTest.java

```diff
@@ -43,69 +53,170 @@
     private static final LocalDateTime HEARING_DATE = LocalDateTime.of(2021, 2, 22, 0, 0, 0).plusMonths(3);
     private static final String RESPONDENT_LAST_NAME = "Smith";
     private static final String CHILD_LAST_NAME = "Jones";
+    private static final UUID REPRESENTATIVE_UUID = UUID.randomUUID();
+    private static final CaseData caseData = buildCaseData();
+    private static final CaseData caseDataBefore = buildCaseDataBefore();
+    private static final EmailContent EMAIL_CONTENT_DOC_NAMES = buildEmailContentDocumentNames();
+    private static final EmailContent EMAIL_CONTENT_NO_DOC_NAMES = buildEmailContentNoDocumentNames();
 
     @Autowired
     private FurtherEvidenceUploadedEventHandler underTest;
 
+    @MockBean
+    private SendDocumentService sendDocumentService;
+
+    @MockBean
+    private CaseAccessDataStoreApi caseAccessDataStoreApi;
+
+    @MockBean
+    private FeatureToggleService featureToggleService;
+
     @Test
-    void sendNotification() {
-        UUID representativeUUID = UUID.randomUUID();
+    void sendNotificationWhenFeatureToggleOffForLAUser() {
```

Comment:
I think we could make the test names a bit more descriptive in this class by trying to mention the toggle in the name, maybe something such as`sendNotificationWhenNewDocumentUploadNotificationToggledOffForLA`. The **AdditionalApplicationsUploadedEventHandlerTest** has some examples of something similar.

What do you think of the idea?

Reply:
Thank you for the comment! Yes, sounds good to me – I will update the test name :)